addons/medical_statemachine/functions/fnc_enteredStateDeath.sqf

```diff
@@ -19,4 +19,7 @@ params ["_unit"];
 // Send a local event before death
 [QEGVAR(medical,death), [_unit]] call CBA_fnc_localEvent;
 
+_unit setVariable [VAR_HEART_RATE, 0, true];
```

Comment:
This should be done in the medical vitals component through an eventhandler I think.

Reply:
Well spotted, I would agree that an EH is better here

Reply:
Would we also want to change leftStateCardiacArrest then?

Reply:
We probably also want to do it for that function, yes. That may require introducing some new events though. Does not have to be done in this PR, imo. But we should create an issue for it if we do not. I will leave that to your discretion @redbery.

Reply:
I don't think I'll have the time to do that today as it is somewhat involved which would mean it would need to wait until next weekend. Think it is better to create a new issue for it if you don't mind.